packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -149,37 +150,47 @@ export class H5GroveApi extends DataProviderApi {
 
   private async processEntityResponse(
     path: string,
-    response: H5GroveEntityResponse
-  ): Promise<Entity> {
-    const { name } = response;
+    response: H5GroveEntityResponse,
+    isChild: true
+  ): Promise<ChildEntity>;
+
+  private async processEntityResponse(
+    path: string,
+    response: H5GroveEntityResponse,
+    isChild?: false
+  ): Promise<ProvidedEntity>;
```

Comment:
I have to add an `isChild` parameter so I can declare signature overrides like in the other provider's APIs. The overrides ensure that when calling `processEntityResponse` without the `isChild` argument in `getEntity`, the return type is `ProvidedEntity` and not just `Entity`.